i18n/translations/ja-JP.json

```
@@ -135,5 +135,16 @@
       "app-context-not-correct-used": "AppContextが正しく使用されませんでした",
       "theme-context-not-correct-used": "ThemeContextが正しく使用されませんでした",
       "package-meta-is-required-at-detail-context": "DetailContextではpackageMetaが必要です"
-  }
-}
\ No newline at end of file
+  },
+  "lng": {
+    "english": "英語",
+    "japanese": "日本語",
+    "portuguese": "ポルトガル語",
+    "spanish": "スペイン語",
+    "german": "ドイツ人",
+    "chinese": "中国の",
```

```suggestion
    "german": "ドイツ語",
    "chinese": "中国語",
```

These lines should fixed like above.